core/executor/util/capabilities.go

```diff
@@ -54,6 +55,7 @@ func DefaultTransmat() rio.Transmat {
 		rio.TransmatKind("gs"):   dirCacher,
 		rio.TransmatKind("file"): fileCacher,
 		rio.TransmatKind("git"):  git.New(filepath.Join(workDir, "git")),
+		rio.TransmatKind("git2"): git2.New(filepath.Join(workDir, "git")),
```

Comment:
Are we confident enough in this that we should flip this around -- make the new code reached by `"git"` and move the old one to `"git-legacy"`? :dancer: